daprdocs/content/en/reference/api/secrets_api.md

```diff
@@ -101,17 +93,11 @@ Code | Description
 ### Examples
 
 ```shell
-curl http://localhost:3500/v1.0/secrets/vault/db-secret
+curl http://localhost:3500/v1.0/secrets/mySecretStore/db-secret
 ```
 
 ```shell
-curl http://localhost:3500/v1.0/secrets/vault/db-secret?metadata.version_id=15&metadata.version_stage=AAA
-```
-
-> Note, in case of deploying into namespace other than  default, the above query will also have to include the namespace metadata (e.g. `production` below)
```

Comment:
TBH, I am completely confused by what was the original intent of this comment. This request doesn't work on vault or k8s secrets AFAIK as neither support `stage_version` . It would work on AWS but then it doesn't support `namespace` query parameter so this whole namespace comment makes no sense. Instead of trying to make sense  of this comment or make it about a hypothetical secret store, I preferred to simply remove it.